api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md

```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
+        </tr>
+        <tr>
+            <td><b>panes</b></td>
+            <td>Array</td>
+            <td>Allows you to customize the automatically-created panes. Each object in this array can contain fields described in the <a href="/Documentation/ApiReference/UI_Components/dxChart/Configuration/panes/" target="_blank">panes</a> property description. When this array holds several objects, use their <a href="/Documentation/ApiReference/UI_Components/dxChart/Configuration/panes/#name" target="_blank">name</a> field to identify the pane.</td>
+        </tr>
+    </table>
+
+    You can also specify any other [Chart properties](/Documentation/ApiReference/UI_Components/dxChart/Configuration/) in this object, except [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource).
+
+- **customizeSeries** 
+
+        <!--JavaScript-->customizeSeries: function (seriesName, seriesOptions) {
+            // Change series properties here
+            return seriesOptions; // This line is optional
+        }
+
+    The **seriesName** argument is a string that allows you to identify a series. The **seriesOptions** argument is an object with the following structure:
+
+    <table class="dx-table full-width">
+        <tr>
+            <th>Field name</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td><b>axis</b></td>
+            <td>String</td>
+            <td>The name of the axis to which the series belongs. Has a value only when the chart has multiple axes.</td>
+        </tr>
+        <tr>
+            <td><b>pane</b></td>
+            <td>String</td>
+            <td>The name of the pane the series occupies. Has a value only when the chart has multiple panes.</td>
+        </tr>
+    </table>
+
+    You can also specify any other [series properties](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) in this object, except [seriesTemplate](Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/).
+
+- **processCell**
+
+        <!--JavaScript-->processCell: function (cellData) {
+            // Process data here
+            return cellData; // This line is optional
+        }
+        
+    The **cellData** object has the following structure:
+
+    <table class="dx-table full-width">
+        <tr>
+            <th>Field name</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td><b>rowPath</b></td>
+            <td>Array</td>
+            <td>The path to the data cell by row field values.</td>
```

Comment:
Do multiple field values correspond to one data cell? Or do you mean:

`The path to the data cell by row field value.`

Reply:
No, there can be several field values that form the path.

Reply:
Okay, thank you.